modules/index-service/src/main/java/org/opencastproject/index/service/impl/IndexServiceImpl.java

```diff
@@ -1629,19 +1629,17 @@ public MediaPackage getEventMediapackage(Event event) throws IndexServiceExcepti
    */
   @Override
   public Source getEventSource(Event event) {
-    if (event.getWorkflowId() != null && isWorkflowActive(event.getWorkflowState()))
+    if (event.getWorkflowId() != null && isWorkflowActive(event.getWorkflowState())) {
       return Source.WORKFLOW;
-
-    if (event.getSchedulingStatus() != null && !event.hasRecordingStarted())
+    } else if (isSchedulerActive(event.getRecordingStatus())) {
```

Comment:
@lkiesow Currently, this is 

```
  public boolean isSchedulerActive(String recordingState) {
    return (recordingState != null) && !RecordingState.UPLOAD_FINISHED.equals(recordingState);
  }
```

Not 100% sure whether any of the KNOWN_STATES would be a better choice.